crates/task-impls/src/quorum_vote.rs

```diff
@@ -487,22 +487,6 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumVoteTaskState<TYPES, I
                     return;
                 }
             }
-            HotShotEvent::ViewChange(new_view) => {
```

Comment:
@shenkeyao This was erroneously killing polling for views since the latest voted view operates distinctly from `cur_view`. It appears as well that this doesn't actually update any state? Since we left ViewChange alive in consensus, I removed this to fix the web tests failing. Now everything seems to be working.